mkdocs.yml

```diff
@@ -47,6 +47,7 @@ nav:
   - Exceptions: 'exceptions.md'
   - Configuration: 'config.md'
   - Test Client: 'testclient.md'
+  - Sync Threadpools: 'threadpools.md'
```

Comment:
I think it's not that easy to find this in the docs, if you don't know that you should look for the threadpool docs.

Reply:
Agreed but can't think of a better name right now. @Kludex any ideas?

Reply:
Concurrency?

Reply:
Concurrency.

Reply:
"Sync" threadpools, as opposed to what?